circle.yml

```diff
@@ -1,4 +1,6 @@
 machine:
+  pre:
+    - curl -sSL https://s3.amazonaws.com/circle-downloads/install-circleci-docker.sh | bash -s -- 1.10.0
```

Comment:
yes, we should probably roll back to the docker pull solution, and manually update nipype/base